constants.js

```diff
@@ -48,7 +51,8 @@ export const titles = [
   {label: 'Tech Leader', minPoints: 88, maxPoints: 115},
   {label: 'Engineering Manager', minPoints: 88, maxPoints: 115},
   {label: 'Software Architect', minPoints: 116},
-  {label: 'Head of Engineering', minPoints: 116}
+  {label: 'Head of Engineering', minPoints: 116},
+  {label: 'Product Manager', minPoints: 130 }
```

Comment:
I'm not sure a Product Manager is part of an Engineer track... We need to think about it, if it's something we see as an organic growth of a developer

Reply:
I'm going to turn into a product manager?!

![giphy](https://user-images.githubusercontent.com/146040/57069553-397f7900-6cd5-11e9-92f8-f06ed3d768ce.gif)

Reply:
> I'm not sure a Product Manager is part of an Engineer track... We need to think about it, if it's something we see as an organic growth of a developer

Not sure what you mean here. I can clarify a bit maybe what I think.

A product manager is someone who understands and manages the complexity of building products in a given market. To do so, they need to understand the technologies involved, marketing strategies, the opportunity and potential for growth or scale.

In software market, of which ours is one in big part, the products are mainly software solutions and understanding the technology deeply enough to make proper decisions puts a fair amount of software engineering knowledge as a hard requirement for a successful product manager.

Since, this snowflake has covers already all technical/engineering skills, without the complementary skills for a successful product manager in such market, for example the ones I added but there could be others which may be as well more important than those, this snowflake will exclude those engineers with the potential of becoming a product manager.

When Medium develops this concept, they do not include product related stuff altogether because they clearly try to separate engineering, more specifically software engineering, from the rest of their stack, because the market their product is not a software (as in to be used by other developers) but rather an "essay authoring tool". So this snowflake suits their software department well enough, while it wouldn't suit ours imo should we want to think of some of our software components as products themselves that address technology market.

That's my reasoning, perhaps I should've added it before. Does it make any sense to you?

Reply:
Of course that is not to say that we will not have products that are not software components, nor that product managers without software engineering knowledge do not have a place.

It is rather to say that, for some of our potential products, say Wikibase if taken as and integrable solution for backing a collaborative knowledge systems, a product manager with such knowledge will probably be key to have.

In that case, I expect those product managers to previous engineers, who as have enough experience in product development as well.

Reply:
sorry about replying just now to your comment.. apparently Jeroen's cat made me blind to it.